agent/discover/cluster.go

```diff
@@ -15,19 +15,25 @@ import (
 
 const crm_monPath string = "/usr/sbin/crm_mon"
 const cib_admPath string = "/usr/sbin/cibadmin"
+const ClusterDiscoverId string = "discover_cluster"
 
 //const crm_monPath string = "./test/fake/crm_mon.sh"
 //const cib_admPath string = "./test/fake_cibadmin.sh"
 
 // This Discover handles any Pacemaker Cluster type
 type ClusterDiscover struct {
+	Id             string
 	host           Discover
 	cibConfig      cib.Root
 	clusterName    string
 	stonithEnabled bool
 	stonithType    consul.ClusterStonithType
 }
 
+func (cluster ClusterDiscover) GetId() string {
+	return cluster.Id
```

Comment:
so its not a uuid, what is it then?

Reply:
This is the `consul` CheckId. As we have more discovery entries with the SAP discovery, the id must belong to the discovery mechanism itself.

Anyway, as commented in other point, I'm not sure if we should include the discovery mechanisms in the consul checks